_docs/best-practices.md

```diff
@@ -185,6 +185,38 @@ s:dateCreated: "2016-12-13"
 s:license: https://www.apache.org/licenses/LICENSE-2.0
 ```
 
+For WDL descriptors, see the [WDL documentation](https://software.broadinstitute.org/wdl/documentation/spec#metadata-section) for how to define metadata.  
+
+Additionally, this following example includes author, email, and description metadata:
+```
+task runtime_meta {
+  String memory_mb
+  String sample_id
+  String param
+  String sample_id
+
+  command {
+    java -Xmx${memory_mb}M -jar task.jar -id ${sample_id} -param ${param} -out ${sample_id}.out
+  }
+  output {
+    File results = "${sample_id}.out"
+  }
+  runtime {
+    docker: "broadinstitute/baseimg"
+  }
+  parameter_meta {
+    memory_mb: "Amount of memory to allocate to the JVM"
+    param: "Some arbitrary parameter"
+    sample_id: "The ID of the sample in format foo_bar_baz"
+  }
+  meta {
+    author: "Denis Yuen"
+    email: "dyuen@oicr.on.ca"
```

Comment:
oh right, we're using live emails in this tutorial. 
Replace the brian emails to "@not-ucsc.org" and mine to "@not-oicr.ca"